### PR TITLE
[Docs] Adding information about using 'htpasswd' for basic-auth's hashing

### PR DESCRIPTION
**Pull request by DaNnNiS**
## Description

Adding information about using htpasswd for hashing of basic-authorization.

## Why do we need it, and what problem does it solve?

Users don't know, that basic authorization is hashed using htpasswd. It's for full information for users.

## What is the expected result?

Users will know about using of htpasswd for basic-auth's hashing.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [x] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

```changes
section: docs
type: fix
summary: Add information about using 'htpasswd' for basic-auth's hashing.
impact_level: low.
```

### CODE REVIEW

**Review by z9r5 — APPROVED** [review]

**Comment by z9r5** [comment]
@nabokihms Please, make a review if you can.

**Review by nabokihms — APPROVED** [review]

File: modules/300-prometheus/docs/USAGE.md [hunk 1]
```diff
@@ -115,6 +115,7 @@ metadata:
   namespace: d8-monitoring
 type: Opaque
 data:
+  #Basic-authorization is hashed using "htpasswd".
```
Comment (z9r5) [inline modules/300-prometheus/docs/USAGE.md:118]:
```suggestion
  # Basic-authorization is hashed using "htpasswd".
```

File: modules/300-prometheus/docs/USAGE_RU.md [hunk 2]
```diff
@@ -115,6 +115,7 @@ metadata:
   namespace: d8-monitoring
 type: Opaque
 data:
+  #Basic-авторизация хешируется с помощью "htpasswd".
```
Comment (z9r5) [inline modules/300-prometheus/docs/USAGE_RU.md:118]:
```suggestion
  # Basic-авторизация хешируется с помощью "htpasswd".
```

**Review by z9r5 — APPROVED** [review]

File: modules/300-prometheus/docs/USAGE.md [hunk 3]
```diff
@@ -115,6 +115,7 @@ metadata:
   namespace: d8-monitoring
 type: Opaque
 data:
+  #Basic-auth is hashed using "htpasswd"
```
Comment (z9r5) [inline modules/300-prometheus/docs/USAGE.md:118]:
```suggestion
  # Basic-auth is hashed using "htpasswd".
```

File: modules/300-prometheus/docs/USAGE_RU.md [hunk 4]
```diff
@@ -115,6 +115,7 @@ metadata:
   namespace: d8-monitoring
 type: Opaque
 data:
+  #Basic-auth хешируется с помощью "htpasswd"
```
Comment (z9r5) [inline modules/300-prometheus/docs/USAGE_RU.md:118]:
```suggestion
  # Basic-auth-строка хешируется с помощью htpasswd.
```

File: modules/300-prometheus/docs/USAGE_RU.md [hunk 5]
```diff
@@ -115,6 +115,7 @@ metadata:
   namespace: d8-monitoring
 type: Opaque
 data:
+  # Basic-auth-строка хешируется с помощью htpasswd.
```
Comment (z9r5) [inline modules/300-prometheus/docs/USAGE_RU.md:118]:
```suggestion
  # Строка basic-auth хешируется с помощью htpasswd.
```

**Pull request merged**